packages/phone-number-privacy/signer/src/common/database/wrappers/request.ts

```diff
@@ -51,15 +40,14 @@ export async function storeRequest(
   logger: Logger,
   trx: Knex.Transaction
 ): Promise<void> {
-  const storeRequestMeter = Histograms.dbOpsInstrumentation.labels('storeRequest').startTimer()
-  logger.debug({ request }, 'Storing salt request')
-  try {
-    await requests(db).transacting(trx).insert(new Request(request)).timeout(DB_TIMEOUT)
-  } catch (err) {
-    Counters.databaseErrors.labels(Labels.update).inc()
-    logger.error({ err }, ErrorMessage.DATABASE_UPDATE_FAILURE)
-    throw err
-  } finally {
-    storeRequestMeter()
-  }
+  return meter(
+    async () => {
+      logger.debug({ request }, 'Storing salt request')
+      await requests(db).transacting(trx).insert(new Request(request)).timeout(DB_TIMEOUT)
+    },
+    [],
+    (err: any) => countAndThrowDBError(err, logger, ErrorMessage.DATABASE_INSERT_FAILURE),
```

Comment:
note: this is different than before but intentionally want this registered as an INSERT failure (not update) to match other behavior